connect-inject/container_init_test.go

```diff
@@ -1508,12 +1507,12 @@ func TestHandlerContainerInit_Resources(t *testing.T) {
 	require.NoError(err)
 	require.Equal(corev1.ResourceRequirements{
 		Limits: corev1.ResourceList{
-			corev1.ResourceCPU:    resource.MustParse(initContainerCPULimit),
-			corev1.ResourceMemory: resource.MustParse(initContainerMemoryLimit),
+			corev1.ResourceCPU:    h.InitCopyContainerCPULimit,
```

Comment:
Because the construction of `h` is empty (`h := Handler{}`), this is asserting it's set to the 0 value. I think a better test would be to instantiate h with these values set to something, e.g.
```go
h := Handler {
  InitCopyContainerCPULimit: resource.MustParse("10m")
  ...
```